pyro/contrib/gp/kernels/kernel.py

```diff
@@ -52,3 +66,267 @@ def _slice_input(self, X):
             return X.unsqueeze(1)
         else:
             raise ValueError("Input X must be either 1 or 2 dimensional.")
+
+    # def __add__(self, other):
+    #     return self.add(other)
+    #
+    # def __mul__(self, other):
+    #     return self.mul(other)
+    #
+    # def __radd__(self, other):
+    #     return self.add(other)
+    #
+    # def __rmul__(self, other):
+    #     return self.mul(other)
+
+    def add(self, other, name=None):
```

Comment:
I'd say go ahead and remove these methods as well.

Reply:
Ya, I get your point!